content/src/service/synchronization/batchDeployer.ts

```diff
@@ -42,6 +45,11 @@ export function createBatchDeployerComponent(
     // ignore entities if those were successfully deployed during this execution
     if (successfulDeployments.has(entity.entityId)) return true
 
+    // ignore specific entity types using EnvironmentConfig.SYNC_IGNORED_ENTITY_TYPES
+    if (syncOptions.ignoredTypes.has(entity.entityType)) {
```

Comment:
Can this be the first check? As it seems like the less expensive

Reply:
yes, makes sense